### DISCUSSION TITLE
Android: Probelm with expansions. Are OBB dismissed after ABB became mandatory?

### DISCUSSION THREAD

**Discussion by PippoApps** [airsdk/Adobe-Runtime-Support#2405] [Q&A]
I have created an OBB and spent some time also with the Distriqt extension.
I do not appear to be able to upload any OBB in Play Console.
The information on google decumentation is probably outdated since it refers to drop downs that do not exist.
Reading around, it seems OBB have been ditched after switching to mandatory ABB (Instead of APK).
Do you have ny suggestion?

**Comment by marchbold** [comment]
Hi @PippoApps , We have communicated in the private repository but just for anyone else coming across this, you should now be using the play asset delivery to package additional assets with your app. See the Harman extension here: https://github.com/airsdk/ANE-PlayAssetDelivery/

**Comment by Oldes** [comment]
@marchbold I am tired of keeping our old games using OBBs alive with all the deprecated Google's code... Do I understand it well, that I could use the mentioned ANE and start using the assets instead of OBBs? Does anybody used it successfully?

**Reply by marchbold** [reply]
Yes you should be using play asset delivery now.

**Reply by Oldes** [reply]
Our games have around 1GB of size... if I drop the existing OBB support, all installed apps will be forced to download new assets, right? May I set the minimum API version to the one which we do not support yet to avoid it? To be honest, it is very unlikely we would do any other updates to these old games except these forced by Google. I just don't want to get myself into some horror situation when such a radical change will break multiple live games.

**Reply by marchbold** [reply]
Hmm yeah not really sure how the change over process would work, but I'd expect as you update to the new system user's will have to download the new assets.
